docs/getting-started.md

```diff
@@ -112,9 +112,22 @@ Hello
 
 ## Setting Up VSCode
 Effekt comes with a basic language server implementation (LSP).
-The [effekt-vscode repository](https://github.com/effekt-lang/effekt-vscode)
-The extension is currently not published, so you will have to install it
-manually by downloading a `vsix` file.
+Source code: [effekt-vscode repository](https://github.com/effekt-lang/effekt-vscode).
+
+### Visual Studio Marketplace 
+
+You can install the extension from the Visual Studio Marketplace.
+
+1. Select `Preferences / Extensions` in the menu
+2. Type "effekt" in the search bar
+3. Click `Install`
```

Comment:
Could we specify the name of the official extension in more detail to deter any name squatting/phishing attempts?The identifier is `effekt-lang.effekt` -- could we perhaps mention it?

Also, side note, is there something like automatic detection of file extensions? (When you open an `.effekt` file, does VSCode automatically recommend you the extension already?)

Reply:
Added the identifier. Don't know about the automatic detection, we should maybe create a separate issue for that.